SitesearchPlugin.inc.php

```diff
@@ -89,17 +84,14 @@ function filterFormAction($output, $templateMgr) {
 	/**
 	 * @see PKPPlugin::getTemplatePath()
 	 */
-	function getTemplatePath($inCore = false) {
+	public function getTemplatePath($inCore = false) {
```

Comment:
The remaining content of this method smells suspiciously like backwards compatibility logic.  `$this->getTemplatePath()` is not called in this plugin.  I suspect the whole method can be deleted in favor of just inheriting the parent implementation.